packages/graphql/src/mutations/marketplace/acceptOffer.js

```diff
@@ -12,10 +12,11 @@ async function acceptOffer(_, data) {
   })
   const { listingId, offerId, marketplace } = parseId(data.offerID, contracts)
 
-  const offer = await marketplace.eventSource.getOffer(listingId, offerId)
-  if (!offer.valid) {
-    throw new Error(`Invalid offer: ${offer.validationError}`)
-  }
+  // Disable for now as this is a heavy call...
+  // const offer = await marketplace.eventSource.getOffer(listingId, offerId)
+  // if (!offer.valid) {
+  //   throw new Error(`Invalid offer: ${offer.validationError}`)
+  // }
```

Comment:
Could move this to that `remoteQuery` thing I added.  Probably not that important though.